src/embeds/TwitchEmbed.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const BaseEmbed = require('./BaseEmbed.js');
+
+/**
+ * Generates a twitch go-live embed
+ */
+class TwitchEmbed extends BaseEmbed {
+  /**
+   * @param {Object} streamData - a stream result from twitch api
+   * @param {Object} userData   - a user result from twitch api
+   */
+  constructor(streamData, userData) {
```

Comment:
I can go add it afterwards, but could you add the stream preview as the attachment (main body image) and the author field as the streamer with their pfp? (and potentially move the small thumbnail to just be that?)

Reply:
I think I understood all of that, except "pfp", what's that?

Reply:
oh, profile picture, sorry.